lib/features/support/capabilities/capabilities.rb

```diff
@@ -5,24 +5,27 @@ class Capabilities
   class << self
     # @param device_type [String] A key from @see Devices::DEVICE_HASH
     # @param local_id [String] unique key for the tunnel instance
-    def for_browser_stack(device_type, local_id, appium_version)
+    # @param capabilties_option [String] extra capabilities provided on the command line
+    def for_browser_stack(device_type, local_id, appium_version, capabilties_option)
       capabilities = {
         'browserstack.console' => 'errors',
         'browserstack.localIdentifier' => local_id,
         'browserstack.local' => 'true',
         'browserstack.networkLogs' => 'true'
       }
       capabilities.merge! Devices::DEVICE_HASH[device_type]
+      capabilities.merge! JSON.parse(capabilties_option)
```

Comment:
nit: the command line option and map are spelled correctly, the option is not


```suggestion
    # @param capabilties_option [String] extra capabilities provided on the command line
    def for_browser_stack(device_type, local_id, appium_version, capabilities_option)
      capabilities = {
        'browserstack.console' => 'errors',
        'browserstack.localIdentifier' => local_id,
        'browserstack.local' => 'true',
        'browserstack.networkLogs' => 'true'
      }
      capabilities.merge! Devices::DEVICE_HASH[device_type]
      capabilities.merge! JSON.parse(capabilities_option)
```

Reply:
I happened to notice the same as I worked on the next branch, where it'll be fixed.